server/src/controllers/Events/resolver.ts

```diff
@@ -348,12 +348,17 @@ export class EventResolver {
       },
     });
 
-    // TODO: find by permission, not role.
     const chapterAdministrators = await prisma.chapter_users.findMany({
       where: {
         chapter_id: chapterId,
         subscribed: true,
-        chapter_role: { is: { name: 'administrator' } },
+        chapter_role: {
+          chapter_role_permissions: {
+            some: {
+              chapter_permission: { name: 'chapter-edit' },
+            },
+          },
```

Comment:
random question, I haven't tested it yet 😞, does this get the instances_owners too, they should have `chapter-edit` as well.

Do we need to add `NOT` that clear the owners 🤔

Reply:
As queried is `chapter_users` table, it will not include instance owner. Unless they have explicitly, for specific chapter, chapter role with `chapter-edit` permission.